webserver/__init__.py

```diff
@@ -20,24 +31,30 @@ def create_app_with_configuration(config_path=None):
     )
 
     # Configuration
-    root_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "..")
-    app.config.from_pyfile(os.path.join(
-        root_path,
-        'default_config.py'
-    ))
-    app.config.from_pyfile(os.path.join(
-        root_path,
-        'custom_config.py'
-    ), silent=True)
+    config_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "config.py")
```

Comment:
what's the config file in production, and how is it loaded? I don't see any `app.config.from_*` that loads consul_config.py, which I understand is the name of the production config file.

Reply:
got it, the consul config file reads from `consul_config.py.ctmpl` and writes to `config.py`